### PR TITLE
[SER-382] Import Platform Unity_States binary

### PR DESCRIPTION
**Pull request by tomthecarrot**

### CODE REVIEW

File: demos/unity_states/unity/Assets/Plugins/unity_states [hunk 1]
```diff
@@ -0,0 +1 @@
+../../../cs/bin/Debug/netstandard2.1
```
Comment (tomthecarrot) [inline demos/unity_states/unity/Assets/Plugins/unity_states:1]: This is a symlink

**Comment by TheButlah** [comment]
Stamping without real review in the interest of time, and because I don't know unity and will have to experiment on monday.

**Review by TheButlah — CHANGES_REQUESTED** [review]
edit: dll needs to be gitignored

**Comment by tomthecarrot** [comment]
> edit: dll needs to be gitignored

This DLL was intentionally checked in since it's not in the symlinked directory. Will symlink just the DLL and re-request.

**Review by TheButlah — APPROVED** [review]

**Pull request merged**